onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1415,193 +1415,728 @@ def _where_input_wrangler(
         torch.float32,
         torch.float16,
     ),
-    "baddbmm": (torch.float32,),
-    "bmm": (torch.float32,),
-    "broadcast_to": (torch.float32,),
-    "cat": (torch.float32,),
-    "ceil": (torch.float32,),
-    "chunk": (torch.float32,),
-    "clamp": (torch.float32,),
-    "clamp_max": (torch.float32,),
-    "clamp_min": (torch.float32,),
-    "clone": (torch.float32,),
-    "col2im": (torch.float32,),
-    "constant_pad_nd": (torch.float32,),
-    "contiguous": (torch.float32,),
-    "convolution": (torch.float32,),
+    "baddbmm": (
+        torch.float32,
+        torch.float16,
+    ),
+    "bmm": (
+        torch.float32,
+        torch.float16,
+    ),
+    "broadcast_to": (
+        torch.float32,
+        torch.float16,
+    ),
+    "cat": (
+        torch.float32,
+        torch.float16,
+    ),
+    "ceil": (
+        torch.float32,
+        torch.float16,
+    ),
+    "chunk": (
+        torch.float32,
+        # torch.float16,  # FIXME: SplitToSequence op inference failed
```

Comment:
nit: Could we link the issue url here? If no, could you please open one?

Reply:
This is just for remembering this issue, will try to fix it later. Otherwise, will file a bug to ORT.